apps/prairielearn/src/pages/shared/syncHelpers.js

```diff
@@ -21,310 +22,121 @@ const error = require('@prairielearn/error');
 
 const docker = new Docker();
 
-module.exports.pullAndUpdate = function (locals, callback) {
-  const options = {
-    course_id: locals.course.id,
-    user_id: locals.user.user_id,
-    authn_user_id: locals.authz_data.authn_user.user_id,
+module.exports.pullAndUpdate = async function (locals) {
```

Comment:
This function specifically serves as a great example of how much more concise and readable new-style server jobs can be! Careful review here would be appreciated since this is such a foundational bit of functionality.

Reply:
It's crazy how much shorter and clearer this now is!! 🚀 

I checked it carefully and it looks good to me.